content/post/data-warehouse-sla.md

```diff
@@ -0,0 +1,100 @@
+---
+author: Scott Breitenother
+title: "Should Your Data Warehouse Have an SLA?"
+cover: "/img/data-warehouse-sla.jpg"
+publishDate: "2018-05-27"
+draft: true
+tags: 
+ - analytics
+ - data warehouse
+ - SLA
+---
+
+Yes, if you want to build a truly data-driven organization your data warehouse needs an SLA. At the core of any data driven organization is trust - your stakeholders must trust that when they need data, it will be there and it will be accurate. Without trust in the data warehouse, your organization will be less likely to use data to drive decisions big and small.
```

Comment:
It might be a good idea to do the "Service level agreement (SLA)" here in the first sentence so that no readers are left behind on your intro